pvnet/training.py

```diff
@@ -121,6 +121,14 @@ def train(config: DictConfig) -> Optional[float]:
 
                 assert os.path.isfile(data_config), f"Data config file not found: {data_config}"
                 shutil.copyfile(data_config, f"{callback.dirpath}/data_config.yaml")
+
+                # upload configuration up to wandb
+                os.makedirs("./configuration", exist_ok=True)
+                shutil.copyfile(data_config, "./configuration/data_config.yaml")
+                OmegaConf.save(config, "./configuration/config.yaml")
+                wandb_logger.experiment.save("./configuration/data_config.yaml")
```

Comment:
Could we not just upload `f"{callback.dirpath}/model_config.yaml"` and `f"{config.datamodule.batch_dir}/data_configuration.yaml"` since these already exist rather than saving them to `./configuration` first?

Reply:
The file structure in wandb matches the file strcutre of the files uploaded, and I wanted it neat, rather than some `mnt/storage4tb_b/..../` for the data configuration. But perhaps there is a neater way to do it?

Reply:
I tried running this:
```
logger.experiment.save("/home/jamesfulton/repos/PVNet_summation/checkpoints/znoi18se/model_config.yaml")
```
and the file appears [here](https://wandb.ai/openclimatefix/pvnet_summation/runs/y745prto/files?nw=nwuserjames_ocf) without any subdirectories. So that suggests `logger.experiment.save(f"{callback.dirpath}/model_config.yaml")` should work and save without subdirs? 

But maybe I'm missing something, and if that doesn't work can we just delete `./configuration` after uploading to wandb to keep things tidy?

Reply:
hmm, yea, ok I could try that. 
How do you feel about override the current `config.yaml` in wandb with a new one? My feeling is that we should save both

Reply:
Yeh I'd save both

Reply:
It looks like I have to use `wandb_logger.experiment.save(f"{callback.dirpath}/data_config.yaml", callback.dirpath)` to make sure its not in a sub folder. Im on a slightly high wandb version that you @dfulu

Reply:
For example https://wandb.ai/openclimatefix/wandb_test/runs/b98z16i9/files?nw=nwuserpeterdudfield